src/peer/dataConnection.js

```diff
@@ -296,6 +305,25 @@ class DataConnection extends Connection {
     }
   }
 
+  /**
+   * Check dcInit argument is valid to enable unreliable mode.
+   * See https://www.w3.org/TR/webrtc/#dom-rtcdatachannelinit
+   * @param {Object} [dcInit] - Config object works as a RTCDataChannelInit.
+   * @return {boolean} Returns this dcInit has valid properties to enable unreliable mode.
+   */
+  _isUnreliableDCInit(dcInit) {
+    if (!dcInit) {
+      return false;
+    }
+
+    // Either of these props are passed, works on unreliable mode.
+    if ('maxRetransmits' in dcInit || 'maxPacketLifeTime' in dcInit) {
```

Comment:
hmmm....
technically we should check if these values are `null` since if they are defined but `null`, it means the dc is reliable.

Do you think that's too much?

Reply:
Unfortunately...

```js
const pc = new RTCPeerConnection();
const dc = pc.createDataChannel('xxx', {
  maxRetransmits: null,
});
dc.reliable; // false in Chrome, true in Firefox
```

So I think it's too much.(= handle these difference correctly is too heavy...